README.md

```diff
@@ -34,7 +34,7 @@ export DISTDIR=/local-scratch/nigam/distdir
 Run the following:
 
 ```
-conda create -n PITON_ENV python=3.10 bazel=5.3 clangxx=14 -c conda-forge
+conda create -n PITON_ENV python=3.10 bazel=5.3 clangxx=14 pathos=0.3.0 -c conda-forge
```

Comment:
What is pathos for?

Reply:
**pathos** is just a more robust version of **multiprocessing** that includes **dill** as its pickle module by default. 

**dill** is just a more robust version of **pickle** that allows us to pickle classes (in this case, Labelers) that have more "exotic" contents, e.g. arguments that are lambdas, certain properties within the class that aren't naturally pickelable, etc.

It's a super minimal change code-wise -- literally just change `multiprocessing.get_context("forkserver").Pool(num_threads)` to `pathos.ProcessPool`

Reply:
I added it to the `pyproject.toml` as well, so I can remove it from this command line command if that's redundant (I'm still not quite clear conceptually how the dependencies in `pyproject.toml` relate to these conda dependencies / how they're installed?)

Reply:
@Lalaland

Reply:
Ah, that sounds pretty handy and good. Yeah, remove it from here and put in pyproject.toml.

The dependencies here are only here because they cannot be in pyproject.toml (due to not being Python packages).